web/models/skill.ex

```diff
@@ -0,0 +1,55 @@
+defmodule CodeCorps.Skill do
+  use CodeCorps.Web, :model
+  alias Inflex
+
+  schema "skills" do
+    field :title, :string
+    field :description, :string
+    field :original_row, :integer
+    field :slug, :string
+
+    timestamps()
+  end
+
+  @doc """
+  Builds a changeset based on the `struct` and `params`.
+  """
+  def changeset(struct, params \\ %{}) do
+    struct
+    |> cast(params, [:title, :description, :original_row, :slug])
+    |> update_slug()
+    |> validate_required([:title, :slug])
+    |> validate_exclusion(:slug, reserved_routes)
+    |> unique_constraint(:slug)
+  end
+
+  def update_slug(changeset) do
+    case changeset do
+      %Ecto.Changeset{changes: %{title: title}} ->
+        slug = Inflex.parameterize(title)
+        put_change(changeset, :slug, slug)
+      _ ->
+        changeset
+    end
+
+  end
+
+  def reserved_routes do
```

Comment:
Not sure where we want this stored...but pretty sure it should be in some file with constants